tasks/98_linked-list/linked-list.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Class representing a Cell
+ */
+class Cell {
+  /**
+   * Create a cell
+   * @param {Number} value, value of cell
+   * @param {null} next, next value
+   */
+  constructor({value, next = null}) {
+    this.value = value;
+    this.next = next;
+  }
+}
+/**
+ * Class representing a Linked List
+ */
+class LinkedList {
+  /**
+   * Create empty linked list
+   */
+  constructor() {
+    this.head = null;
+  }
+  /**
+   * Get the size of linked list
+   * @return {Number} size of linked list
+   */
+  get amount() {
+    let result = 0;
```

Comment:
I think you can create a field like `length` and update it on `push` and `pop` and return it here instead of calculating it on every call.